app/services/data/bank-account-details.js

```diff
@@ -1,10 +1,10 @@
 const config = require('../../../knexfile').extweb
 const knex = require('knex')(config)
 
-module.exports.insert = function (claimId, bankDetailsData) {
+module.exports.insert = function (claimId, bankDetailsDetails) {
```

Comment:
as above, rename and only expose this single function, so there is only `module.exports = function (claimId, bankDetailsDetails)`